definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json

```diff
@@ -24,5 +24,18 @@
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/make-service-decision",
     "RetriesTimeoutURLAboutToSubmitEvent": "30,30",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "04/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ServicePayment",
+    "Name": "Service payment exists",
+    "Description": "Service payment exists",
+    "DisplayOrder": 13,
+    "PreConditionState(s)": "AwaitingPayment",
+    "PostConditionState": "*",
```

Comment:
This is supposed to move the case to the other new state, isn't it?

Reply:
Youre correct, but in the testing phase, i needed to be able to try it over and over again. This will be changed when all other work has been completed